Code/Framework/AzCore/AzCore/Serialization/Json/PointerJsonSerializer.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Serialization/Json/PointerJsonSerializer.h>
```

Comment:
The PointerJsonSerializer is missing changes https://github.com/o3de/o3de/commit/3769dbf9a2bba5d0af2600a2a63fe0ffbb0f83b4

Reply:
ok, everything seems to work locally, I think we're good to go 👍